util/regexp-assemble/data/942120.data

```diff
@@ -1,5 +1,5 @@
 ##! Please refer to the documentation at 
-##! https://coreruleset.org/docs/development/regexp_assemble/.
+##! https://coreruleset.org/    docs/development/regexp_assemble/.
```

Comment:
Spaces break the link.

Reply:
Thanks. Probably I fell asleep in the space bar :D